src/dotnet-interactive-vscode-insiders/package.json

```diff
@@ -256,67 +256,67 @@
       {
         "id": "dotnet-interactive.csharp",
         "aliases": [
-          "C# (.NET Interactive)"
+          "C# Script"
         ],
         "configuration": "./syntaxes/csharp.language-configuration.json"
       },
       {
         "id": "dotnet-interactive.fsharp",
         "aliases": [
-          "F# (.NET Interactive)"
+          "F#"

```

Comment:
Can't have this; it clashes with the "real" F# language.  Similar for the others.

Reply:
@jonsequitur  what do we do? `F# (Polyglot Notebook)`? @claudiaregio ??

Reply:
My comment below comes with the following assumptions, please let me know if they're wrong. 
1. This is referring to the language that gets saved in the metadata of the notebook
2. This is what is displayed in the cell status bar in the bottom right of a cell in a VS Code notebook

My personal opinion is that it should be F# (.NET Interactive), C# (.NET Interactive), etc. The reason being that the special version of C#, F# that is being run is being run by .NET Interactive. In addition, if Jupyter one day decides to support a language picker, a Jupyter/JupyterLab user would have no notion of the "Polyglot Notebooks" extension in VS Code since they can just install the .NET Interactive kernel. This way for the Jupyter use case it would also be more clear what is running that C#, F# code etc.

Reply:
What should Python, JavaScript, and so on say? What about different execution environments, e.g. JavaScript in the browser versus nodeJS? 

Regardless of the label, these are all unions of the language in question and the Polyglot Notebook language (which adds magic commands.)

What we're really discussing here is the kernel picker experience generally and maybe we should move the discussion over to that issue.